src/workerd/api/basics.h

```diff
@@ -609,15 +619,56 @@ class AbortSignal final: public EventTarget {
     tracker.trackField("reason", reason);
   }
 
+  void serialize(jsg::Lock& js, jsg::Serializer& serializer);
+
+  // To test what happens if a capability is dropped before invoking release on the cloned abort
+  // signal, this method will tell every rpcClient to skip this step before destruction.
+  void skipReleaseForTest();
+
+  static jsg::Ref<AbortSignal> deserialize(
+      jsg::Lock& js, rpc::SerializationTag tag, jsg::Deserializer& deserializer);
+
+  JSG_SERIALIZABLE(rpc::SerializationTag::ABORT_SIGNAL);
+
  private:
   IoOwn<RefcountedCanceler> canceler;
   Flag flag;
   kj::Maybe<jsg::JsRef<jsg::JsValue>> reason;
   kj::Maybe<jsg::JsRef<jsg::JsValue>> onAbortHandler;
 
+  static kj::Exception abortException(
+      jsg::Lock& js, jsg::Optional<kj::OneOf<kj::Exception, jsg::JsValue>> reason);
+
   void visitForGc(jsg::GcVisitor& visitor);
 
   friend class AbortController;
+
+  /** RPC client functionality. Used if this signal was serialized. **/
```

Comment:
OK this is really nitpicky but we don't use `/*` block comments. "Section header" comments are usually styled like this:

```suggestion
  // -----------------------------------------------------------------
  // RPC client functionality. Used if this signal was serialized.
```